README.md

```diff
@@ -79,6 +79,28 @@ Kerberos support is implemented via the [requests-kerberos](https://github.com/r
 
 Currently, sparkmagic does not support passing a kerberos principal/token, but we welcome pull requests.
```

Comment:
Is this still the case?

Reply:
I wasn't able to confirm it since my environment does not use this configuration, but since the constructor accepts a principal I would say it is ok now